include/rcpputils/filesystem_helper.hpp

```diff
@@ -190,6 +197,28 @@ inline bool create_directories(const path & p)
 
 #undef RCPPUTILS_IMPL_OS_DIRSEP
 
+/**
+ * Remove extension(s) from a path. An extension is defined as text starting from the end of a
+ * path to the first period (.) character.
+ *
+ * \param file_path The file path string.
+ * \param n_times The number of extensions to remove if there are multiple extensions.
+ * \return The path object.
+ */
+path remove_extension(const path & file_path, int n_times = 1)
```

Comment:
```suggestion
inline path remove_extension(const path & file_path, int n_times = 1)
```

Unlike methods, functions in headers needs to be explicitly marked as inline. Otherwise, you may end up breaking the ODR rule.

https://stackoverflow.com/questions/5057021/why-are-c-inline-functions-in-the-header

Reply:
Done